src/index.js

```diff
@@ -4,10 +4,31 @@ import './wdyr';
 
 import React from 'react';
 import ReactDOM from 'react-dom';
+import * as Sentry from '@sentry/react';
+import { Integrations } from '@sentry/tracing';
+import { createBrowserHistory } from 'history';
+
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 
+const history = createBrowserHistory();
+
+const { REACT_APP_SENTRY_DSN, REACT_APP_SENTRY_ENVIRONMENT, REACT_APP_VERSION } = process.env;
+
+if (REACT_APP_SENTRY_DSN !== undefined) {
+  Sentry.init({
+    dsn: REACT_APP_SENTRY_DSN,
+    ...(REACT_APP_SENTRY_ENVIRONMENT && { environment: REACT_APP_SENTRY_ENVIRONMENT }),
+    ...(REACT_APP_VERSION && { release: `GeoCRUD-front@${REACT_APP_VERSION}` }),
```

Comment:
I suggest to include sample Rate management https://docs.sentry.io/platforms/javascript/configuration/sampling/

Reply:
Oh yes, top!
Do you have a suggestion of the best value to take. I don't have enough experience in Sentry to choose one. By default I will copy/paste the example into the doc.

Reply:
it's a range from **0.0** to **1.0**.  0 is disabled, so i suggest to use 0 by default

Reply:
I'm not sure about this value 

> to a number between 0 (0% of errors sent) and 1 (100% of errors sent)

Sentry should become useless if it no longer sends errors?

Reply:
SampleRate is not for error reporting, it's for performance monitoring. 0 -> 0% of transactions are monitored. 1 -> 100% of transactions are monitored. In all case, 0 to 1, errors are reported

Reply:
Oups ! it's not sampleRate, it's tracesSampleRate

Reply:
sampleRate is for error, so 1 by default. tracesSampleRate is for performance monitoring, so 0 by default

Reply:
Done ! 👍